src/storage/store/stores.rs

```diff
@@ -248,9 +248,18 @@ impl Stores {
             let max_messages =
                 self.store_limits
                     .max_messages(slot.units, slot.legacy_units, store_type);
+            let name = match store_type {
```

Comment:
Clean to do a string replace instead of enumerating manually. But not a big deal.